centrallix/netdrivers/net_http_rest.c

```diff
@@ -114,10 +115,28 @@ nht_i_RestWriteAttrValue(pNhtConn conn, pObject obj, char* attrname, int data_ty
 		break;
 
 	    case DATA_T_MONEY:
-		nht_i_QPrintfConn(conn, 0, "{ \"wholepart\":%INT, \"fractionpart\":%INT }",
-			od.Money->WholePart,
-			od.Money->FractionPart
-			);
+	        /** For the time being, 64-bit representation will mimic whole and fraction part handling of negatives
+	         ** with regards to JSON data.  It will use the new %LL specifier, but the JSON returned will
+	         ** contain an inverted fraction part that always counts in the positive direction.
+	         **/
+	        
+	        /** If there is a negative with a non-zero fraction part **/
+	        if (od.Money->Value < 0 && od.Money->Value%10000 != 0)
+                {
+	                whole = (od.Money->Value + 10000)/10000ll;
+	                fraction = 10000ll - od.Money->Value%10000;
```

Comment:
This math doesn't look right.  For example, for value -9999 (-$0.9999), whole should be -1 and fraction should be 1.  For value -10001 (-$1.0001), whole should be -2 and fraction should be 9999.